src/main/app/forms/models/address.ts

```diff
@@ -65,6 +62,6 @@ export class Address implements Serializable<Address> {
   }
 
   toString (): string {
```

Comment:
I know it's not a new method, but will be good to add unit tests for this as well.